tests/fast_tests/test_examples.py

```diff
@@ -184,52 +152,65 @@ def setUp(self):
         if not ray.is_initialized():
             ray.init(num_cpus=1)
 
-    def test_figure_eight(self):
-        alg_run, env_name, config = figure_eight_setup()
-        self.run_exp(alg_run, env_name, config)
+    def test_singleagent_figure_eight(self):
+        self.run_exp(singleagent_figure_eight)
 
     def test_traffic_light_grid(self):
-        # test the example in the absence of inflows
-        alg_run, env_name, config = traffic_light_grid_setup(use_inflows=False)
-        self.run_exp(alg_run, env_name, config)
+        pass  # FIXME
```

Comment:
Do these no longer work?

Reply:
Since traffic light needs extra parameters, we did not include it in this PR. I will open a PR soon for this. All the tests that have pass or FIXME will soon be added in the new PR.